tests/e2e/specs/shopper/cart-checkout/checkout.test.js

```diff
@@ -196,7 +196,7 @@ describe( 'Shopper → Checkout', () => {
 		} );
 	} );
 
-	describe( `Shipping`, () => {
+	describe.only( `Shipping`, () => {
```

Comment:
Look like this is left from debugging.

Reply:
Doh! Thanks for spotting that, remove the only!